sky/packages/sky/lib/src/rendering/custom_layout.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'box.dart';
+import 'object.dart';
+
+class _MultiChildParentData extends ContainerBoxParentDataMixin<RenderBox> { }
+
+abstract class MultiChildLayoutDelegate {
+  Map<Object, RenderBox> _idToChildMap = new Map<Object, RenderBox>();
+  RenderBox _idToChild(Object childId) { return  _idToChildMap[childId]; }
```

Comment:
Why have this function?  Seems like just as easy to write _idToChildMap[foo] as _idToChild(foo)...  Maybe rename _idToChildMap to just _children so it's just _children[foo] ?